client/my-sites/customer-home/cards/tasks/deprecate-editor/index.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import Task from '../task';
+
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import {
+	composeAnalytics,
+	recordGoogleEvent,
+	recordTracksEvent,
+	withAnalytics,
+	bumpStat,
+} from 'state/analytics/actions';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import blockEditorImage from 'assets/images/illustrations/block-editor-fade.svg';
+import FormattedDate from 'components/formatted-date';
+import { localizeUrl } from 'lib/i18n-utils';
+import InlineSupportLink from 'components/inline-support-link';
+
+const DeprecateEditor = ( { siteId, gutenbergUrl, optIn } ) => {
+	const translate = useTranslate();
+	const actionCallback = () => {
+		optIn( siteId, gutenbergUrl );
+	};
+	return (
+		<Task
+			title={ translate( 'The Block Editor is coming.' ) }
+			description={ preventWidows(
+				translate(
+					'Get a head start before we enable it for everyone on {{strong}}{{date}}{{/date}}{{/strong}}. {{support}}{{/support}}',
+					{
+						components: {
+							strong: <strong />,
+							date: <FormattedDate date="2020-06-01" format="MMMM D" />,
+							support: (
+								<InlineSupportLink
+									supportPostId={ 165338 }
+									supportLink={ localizeUrl(
+										'https://wordpress.com/support/block-editor-is-coming'
+									) }
+									showIcon={ false }
+									text={ translate( 'Read more' ) }
+									tracksEvent="calypso_customer_home_editor_deprecate_support_page_view"
+									statsGroup="calypso_customer_home"
+									statsName="editor_deprecate_learn_more"
+								/>
+							),
+						},
+					}
+				)
+			) }
+			actionText={ translate( 'Switch to the Block Editor' ) }
+			actionOnClick={ actionCallback }
+			illustration={ blockEditorImage }
+			badgeText={ translate( "What's new" ) }
+			taskId="deprecate-editor"
```

Comment:
This needs to match the format of the constant for automagic dismiss handling (it should be `editor-deprecation`).

Reply:
Hmm, I'm thinking we might want to think through string uid mapping to component mapping, in a follow up. We're currently doing this twice for tasks which can lead to typos.

Reply:
Thanks, taskId updated now.